src/Xlang/README.md

```diff
@@ -0,0 +1,19 @@
+## Microsoft/Xlang/UndockedRegFreeWinRT
+
+Do not change code under the UndockedRegFreeWinRT directory; it contains Microsoft/Xlang/UndockedRegFreeWinRT source code. 
+
+It is created using git subtree command and points to this specific commit [cfe510d0d2b07484fea2c6d77163de017738c100]:
+
+    git subtree add --prefix=src/Xlang/UndockedRegFreeWinRT https://github.com/microsoft/xlang master --squash

```

Comment:
>master [](http://example.com/codeflow?start=95&length=6)

cfe510d0d2b07484fea2c6d77163de017738c100

Reply:
Changed to suggested